### ISSUE TITLE
Hacktoberfest: Cypress E2E tests

### ISSUE THREAD

**Issue by FredrikOseberg** [Unleash/unleash-frontend#392]
Want to contribute for hactoberfest? Help us improve our frontend E2E testing. We use a framework called Cypress and would like to extend the E2E testing to more CRUD based operations in our frontend application: 

- Custom strategies CRUD 
- Tag Types CRUD
- API Keys CRUD
- Addons CRUD

Examples on how to implement the tests can be found in: 

`cypress/integration/feature.spec.js`

The tests needs a user to run. Before you begin you should create a test user in the application. Currently the tests expects a user to exists with the following credentials: 

```test@test.com / qY70$NDcJNXA```

After you've set up the test user, try running the tests by running `yarn e2e`.

**Comment by jyoungiv** [comment]
I would like to contribute, may I be assigned?

**Comment by FredrikOseberg** [comment]
> I would like to contribute, may I be assigned?

Certainly. I'll update the Addons (Create, Read, Update, Delete) to in progress. Please let me know if you need any help to get started. If you have any questions or need any input please reach out to me either here or through our slack community: https://join.slack.com/t/unleash-community/shared_invite/zt-8b6l1uut-LL67kLpIXm9bcN3~6RVaRQ

Thanks!

**Comment by FredrikOseberg** [comment]
> Hey @ivarconr , How do I run the unleash-api? I did not find any repo with that name. I'm getting this error without running api on `yarn start`
> 
> <img alt="Screenshot 2021-10-15 at 1 19 21 PM" width="1552" src="https://user-images.githubusercontent.com/3799600/137452140-962dcb64-9d44-471d-8d9c-b9e260e6fede.png">
> 
> I started frontend with heroku using `yarn run start:heroku` but when I run cypress tests using `yarn run e2e`, it throws out an error probably because the password field isn't present on the UI. Any suggestions <img alt="Screenshot 2021-10-15 at 1 26 39 PM" width="1312" src="https://user-images.githubusercontent.com/3799600/137452781-03c875d6-99d9-428b-8e48-36757ef84196.png"> ?

@devcer 

If you want to run the E2E tests against the heroku instance you must add omit the PASSWORD_AUTH=true parameter in the e2e test command. You can try running the same command without this parameter, or you could set up your own local instance of unleash as outlined [here](https://docs.getunleash.io/deploy/getting_started)
